app/models/vm_or_template.rb

```diff
@@ -1884,6 +1884,10 @@ def supports_snapshots?
     false
   end
 
+  def supports_provisioning?
+    true
```

Comment:
@h-kataria @gmcculloug Instead of adding a new method, can we use [vm_or_template.has_active_ems?](https://github.com/ManageIQ/manageiq/blob/472e0d0aad20f288f2f59d02195ed28d35d9f453/app/models/vm_or_template.rb#L1760)